projects/go-style-guide/src/app/features/ui-kit/components/badge-docs/badge-docs.coponent.ts

```diff
@@ -36,4 +36,8 @@ export class BadgeDocsComponent {
   `;
 
   pageTitle: string = 'Badge';
+  // linkToSource(): void {
+  //   window.open('https://github.com/mobi/goponents/tree/dev/projects/go-lib/src/lib/components/go-badge','_blank')
+  // }
```

Comment:
Probably don't need this comment @Ashygoyal 😬

Reply:
aah, my bad ! Removed in latest commit.